components/UpsellPlusModal.vue

```diff
@@ -0,0 +1,229 @@
+<template>
+  <UModal
+    :title="$t('upsell_plus_modal_title')"
+    :ui="{
+      title: 'text-sm laptop:text-base font-bold text-center',
+      footer: 'flex flex-col items-center w-full',
+      body: 'w-full flex flex-col items-start mx-0 laptop:mx-6 gap-2 text-sm laptop:text-base',
```

Comment:
<img width="682" height="669" alt="Image" src="https://github.com/user-attachments/assets/7e6b6495-169a-40b6-9a57-9b1ebb9404f1" />

```suggestion
      body: 'flex flex-col items-start gap-2 max-laptop:text-sm',
```